NEWS.rst

```diff
@@ -2,9 +2,116 @@
+  ready state (1350_)
+* Fixed drop notifications to include as much metadata as possible (1427_, 1444_)
+* Fixed a bug where the compilation of the base programs and writing of header
+  files could occur in parallel with compilation of programs for endpoints which
+  could lead to temporary compilation errors (1440_)
+* Fail gracefully when configuring more than the maximum supported L4 ports in
+  the policy (1406_)
+* Fixed a bug where not all policy rules were JSON validated before sending it
+  to the agent (1406_)
+* Fixed a bug in the SHA256 calculation (1454_)
+* Fixed the datapath to differentiate the packets from a regular local process
+  and packets originating from the proxy (previously redirected to by the
+  datapath). (1459_)
+
+Features
+--------
+* The monitor now supports multiple readers, you can run `cilium monitor`
+  multiple times in parallel. All monitors will see all events. (1288_)
+* `cilium policy trace` can now trace policy decisions based on Kubernetes pod
+  names, security identities, endpoint IDs and Kubernetes YAML resources
+  [Deployments, ReplicaSets, ReplicationControllers, Pods ](1124_)
+* It is now possible to reach the local host on IPs which are within the
+  overall cluster prefix (1394_)
+* The `cilium identity get` CLI and API can now resolve global identities with
+  the help of the kvstore (1313_)
+* Use new probe functionality of LLVM to automatically use new BPF compare
+  instructions if supported by both LLVM and the kernel (1356_)
+* CIDR network policy is now visible in `cilium endpoint get` (1328_)
+* Set minimum amount of compilation workers to 4 (1227_)
+* Removed local backend (1235_)
+* Reduced use of cgo in in bpf packages (1275_)
+* Do sparse checks during BPF compilation (1175_)
+* New `cilium bpf lb list` command (1317_)
+* New optimized kvstore interaction code (1365_, 1397_, 1370_)
+* The access log now includes a SHA hash for each reported label to allow for
+  validation with the kvstore (1425_)
+
+CI
+--
+* Improved CI testing infrastructure (1262_, 1207_, 1380_, 1373_, 1390_, 1385_, 1410_)
+
+
+Documentation
+-------------
+* Multi networking documentation (1244_)
+* Documentation of the policy specification (1344_)
+* New improved top level structuring of the sections (1344_)
+* Example for etcd configuration file (1268_)
+* Tutorial on how to use cilium monitor for troubleshooting (1451_)
+
+Kubernetes
+----------
+* Added support for Custom Resource Definition (CRD). Be aware that parallel
+  usage of CRD and Third party Resources (TPR) leads to unexpected behaviour.
+  See cilium.link/migrate-tpr for more details. Upgrade your
+  CiliumNetworkPolicy resources to cilium.io/v2 in order to use CRD. Keep them
+  at cilium.io/v1 to stay on TPR. (1169_, 1219_)
+* The CiliumNetworkPolicy resource now has a status field which contains the
+  status of each node enforcing the policy (1354_)
+* Added RBAC rules for v1/NetworkPolicy (1188_)
+* Upgraded to kubeadm 1.7.0 (1179_)
```

Comment:
Should this be in CI? Also the link to #1179 doesn't seem to work?

Reply:
I am not able to click on any of the links actually.

Reply:
@manalibhutiyani here https://github.com/cilium/cilium/blob/d45795fd5e9e675cc1b4701d3e2d7c95436c7afc/NEWS.rst

Reply:
thanks!

Reply:
Fixed